### PR TITLE
Metamist api transformation

### PR DESCRIPTION
**Pull request by MattWellie**
# Fixes

  - Updates to use GQL
  - Contains a fix for the Peddy.UNKNOWN problem currently holding up ag-cardiac

## Proposed Changes

  - Includes sequencingGroup changes
  - These gql changes are (almost) all in the helper scripts, run to prepare a workflow rather than during
  - Changes have been tested locally, looks functional
  - Pedigree member sexes are explicitly written down to 'unknown' if they are neither male or female
  - `ag-cardiac` added to the standard runtime config, with Adult Cardiac SuperPanel

## Checklist

- [x] Related Issue created
- [x] Tests covering new change
- [x] Linting checks pass

### CODE REVIEW

File: reanalysis/metamist_registration.py [hunk 1]
```diff
@@ -14,62 +14,14 @@
...
 
-from sample_metadata.apis import AnalysisApi
-from sample_metadata.model.analysis_type import AnalysisType
-from sample_metadata.model.analysis_model import AnalysisModel
-from sample_metadata.model.analysis_status import AnalysisStatus
-from sample_metadata.model.analysis_query_model import AnalysisQueryModel
-from sample_metadata.model.analysis_update_model import AnalysisUpdateModel
-
-
-def inactivate_old_entry(name: str, metadata: dict[str, str], anal_type: AnalysisType):
```
Comment (MattWellie) [inline reanalysis/metamist_registration.py:25]: this inactivation isn't necessary; the active flag in metamist doesn't currently behave correctly so it's a bit pointless

File: reanalysis/utils.py [hunk 2]
```diff
@@ -851,6 +851,8 @@ def get_simple_moi(input_moi: str | None, chrom: str) -> str | None:
         case ['both', *_additional]:
             return 'Mono_And_Biallelic'
         case ['monoallelic', *_additional]:
+            if chrom in X_CHROMOSOME:
```
Comment (MattWellie) [inline reanalysis/utils.py:854]: closes #303

File: setting_up_run.md
Comment (MattWellie) [inline setting_up_run.md:1]: this whole document was way out of date

File: helpers/hpo_panel_matching.py [hunk 3]
```diff
@@ -16,20 +15,17 @@
 
 import networkx
 from obonet import read_obo
+
+# TODO update to graphQL API once it works for Seqr metadata query
```
Comment (MattWellie) [inline helpers/hpo_panel_matching.py:19]: no graphql equivalent endpoint exists for this

**Review by cassimons — APPROVED** [review]

**Pull request merged**